typescript/web/src/components/members/new-member-modal.tsx

```diff
@@ -129,7 +132,7 @@ user2@example.com
                     : ""
                 } ${
                   hasInvalidEmails
-                    ? "\nAt lease one email format is invalid. Write one email address per line."
+                    ? "\nAt lease one email format is invalid. Emails should be separated by , ; or spaces."
```

Comment:
[Thanks to `\s` in the regex ](https://regexone.com/lesson/whitespaces), it's still valid (and more readable with the textbox we use) to use new lines.

Also, there is a typo, it's `At-least`, not `At lease`.

```suggestion
                    ? "\nAt least one email address is invalid. Addresses can be separated them by line-breaks, spaces, commas or semicolons."
```

Reply:
"line-breaks" may be a bit too technical no, maybe "new lines"?

Reply:
@valerian That's what I was thinking too before realising that it's actually line-breaks, not new-lines.
I agree however that I'd be clearer and abusing the `new line` words has the advantage that everyone will understand. Let's use it 👍 🚀